datajunction-server/datajunction_server/api/catalogs.py

```diff
@@ -21,6 +21,8 @@
 settings = get_settings()
 router = SecureAPIRouter(tags=["catalogs"])
 
+UNKNOWN_CATALOG_ID = 0
```

Comment:
Are you sure all the known catalogs get a positive integer id?

Reply:
So at the moment the id is an incremental id set by the database and it starts at 1. This should be true for both SQLite and Postgres. So unless someone is manually creating a catalog entry and setting the id to 0, we should be safe.